<launch>
    <node pkg="tf" type="static_transform_publisher" name="tf_map_uwb" 
          args="0.0 0.0 1.5 0 0 0 map uwb_base_link 100" />

    <node pkg="tf" type="static_transform_publisher" name="tf_anchor_left" 
          args="0.0 0.02857 0 0.6108 0 0 uwb_base_link anchor_left_link 100" />

    <node pkg="tf" type="static_transform_publisher" name="tf_anchor_right" 
          args="0.0 -0.02857 0 -0.6108 0 0 uwb_base_link anchor_right_link 100" />


    <node pkg="uwb_fusion" type="uwb_optimizer_node" name="uwb_optimizer" output="screen">
        
        <param name="topic_uwb_left" value="/uwb/sensor1_data"/>
        <param name="topic_uwb_right" value="/uwb/sensor2_data"/>

        <param name="map_frame" value="map"/>

        <param name="noise_range_sigma" value="0.10"/>
        <param name="noise_angle_sigma" value="10.0"/>  
        <param name="noise_elev_sigma" value="60.0"/>  
        <param name="fov_limit_deg" value="120.0"/> 
        <param name="person_vel_sigma" value="50"/>   
        <param name="person_z_vel_sigma" value="0.1"/> 
        
        <param name="lpf_gain" value="0.1"/> 

        <param name="blind_cov_thresh" value="1.0"/>

    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find uwb_fusion)/config/uwb_test.rviz" />

</launch>